04-microbot.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: microbot
+  namespace: default
+  labels:
+    k8s-app: microbot
+    kubernetes.io/cluster-service: "true"
+    kubernetes.io/name: "microbot"
+spec:
+  selector:
+    k8s-app: microbot
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+  externalIPs:
+  - $EXT_IP1
+    
+
+---
+
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: microbot
+  namespace: default
+  labels:
+    k8s-app: microbot
+    version: v1
+    kubernetes.io/cluster-service: "true"
+spec:
+  replicas: 25
```

Comment:
25 replicas of a very small service to show that it's actually placing the service across the hosts, representing multiple replicas as a service and then exposing those as an externally accessible service. You're welcome to have it to something smaller, but I thought it was a good example.
Thanks for getting the main project started, it came in handy for a setup I was trying out last night.







On Sat, Mar 19, 2016 at 8:39 PM -0700, "Herman Junge" <notifications@github.com> wrote:












In 04-microbot.yaml:
> +  - $EXT_IP1
> +    
> +
> +---
> +
> +apiVersion: v1
> +kind: ReplicationController
> +metadata:
> +  name: microbot
> +  namespace: default
> +  labels:
> +    k8s-app: microbot
> +    version: v1
> +    kubernetes.io/cluster-service: "true"
> +spec:
> +  replicas: 25



Why 25???!!!



—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub